examples/nirfmxgsm/evm.py

```diff
@@ -0,0 +1,249 @@
+r"""EVM example.
+
+Steps:
+1. Open a new RFmx Session.
+2. Configure Frequency Reference.
+3. Configure basic signal properties (Center Frequency, Reference Level and External Attenuation).
+4. Configure Trigger Parameters for IQ Power Edge Trigger.
+5. Configure Number of Timeslots.
+6. Configure Auto tsc Detection Enabled.
+7. Configure Signal Type.
+8. Configure tsc.
+9. Select ModAcc measurement and enable Traces.
+10. Configure Averaging Parameters for ModAcc measurement.
+11. Initiate the Measurement.
+12. Fetch ModAcc Measurements and Traces.
+13. Close RFmx Session.
+
+The gRPC API is built from the C API. RFmx GSM documentation is installed with the driver at:
+C:\Program Files (x86)\National Instruments\RFmx\GSM\Documentation\rfmxgsmcvi.chm
+
+Getting Started:
+
+To run this example, install "RFmx GSM" on the server machine.
+Link: https://www.ni.com/en-us/support/downloads/software-products/download.rfmx-gsm-edge-.html
```

Comment:
Unrelated to these changes, just interesting that the download url ends in a `-`. It's the correct URL just looks weird.

Reply:
Agreed!